v3/tutorials/07-add-a-pallet/index.mdx

```diff
@@ -510,18 +524,20 @@ After the build succeeds, you can start the node:
 
 ### Start the Front-End
 
-As in the previous tutorials, this tutorial will use the Substrate Developer Hub Front-End Template
+As in the previous tutorials, this tutorial will use the
+[Substrate Front-End Template](https://github.com/substrate-developer-hub/substrate-front-end-template)
```

Comment:
Use the monthly tagged template for consistency? Anything older than latest is incompatible AFAIK

Reply:
done.